src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -88,15 +88,15 @@ public function __construct(
      */
     public function storeAllowsPaying($paymentMethodName, $position = null)
     {
-        $this->createPaymentMethod($paymentMethodName, 'PM_'.$paymentMethodName, 'Payment method', true, $position);
+        $this->createPaymentMethod($paymentMethodName, 'PM_'.$paymentMethodName, 'Offline', 'Payment method', true, $position);
     }
 
     /**
      * @Given /^the store allows paying (\w+) for (all channels)$/
      */
     public function storeAllowsPayingForAllChannels($paymentMethodName, array $channels)
     {
-        $paymentMethod = $this->createPaymentMethod($paymentMethodName, StringInflector::nameToUppercaseCode($paymentMethodName), 'Payment method', false);
+        $paymentMethod = $this->createPaymentMethod($paymentMethodName, StringInflector::nameToUppercaseCode($paymentMethodName), 'Offline', 'Payment method', false);
```

Comment:
I think it would be better to keep it, as it not only consist of factory call but also some more logic (setting position, setting to shared storage and to repository).